xsd/src/main/jaxb/bindings.xjb

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <bindings version="2.1"
           xmlns="http://java.sun.com/xml/ns/jaxb"
+		  xmlns:jaxb="http://java.sun.com/xml/ns/jaxb"
           xmlns:xs="http://www.w3.org/2001/XMLSchema"
           xmlns:xjc="http://java.sun.com/xml/ns/jaxb/xjc"
           xmlns:wildcard="http://jaxb2-commons.dev.java.net/basic/wildcard"
           xmlns:inheritance="http://jaxb2-commons.dev.java.net/basic/inheritance"
-          extensionBindingPrefixes="xjc">
+		  jaxb:extensionBindingPrefixes="xjc">
 
 	<globalBindings>
 		<xjc:simple/>
 		<xjc:javaType adapter="no.digipost.xsd.jaxb.XSDateTimeAdapter" name="java.time.ZonedDateTime"
 		              xmlType="xs:dateTime"/>
 		<xjc:javaType adapter="no.digipost.xsd.jaxb.XSDateAdapter" name="java.time.LocalDate"
 		              xmlType="xs:date"/>
+		<jaxb:javaType name="java.time.ZonedDateTime" xmlType="xs:dateTime"
```

Comment:
Gjør vi ikke dette allerede med `XSDateTimeAdapter` definert på linje 13? Hvordan fungerer dette sammen med eksisterende global binding for `java.time.ZonedDateTime` <---> `xs:dateTime` ?

Reply:
Mulig jeg har rotet litt med versjoner av bygg her, men jeg har hatt noen som har vært generert med XmlGregorianCalendar. Jeg dobbeltsjekker og oppdaterer.

Reply:
Du har rett.

Reply:
9f220fe